model-api-gen-gradle-test/typescript-generation/src/node.test.ts

```diff
@@ -0,0 +1,17 @@
+import { useFakeNode } from "./test-helpers";
+
+test("untyped nodes can be removed", () => {
```

Comment:
`model-api-gen-gradle-test/typescript-generation/src/node.test.ts` should test only features of typed nodes.

The features of the untyped nodes should be tested in closer to the implementation in `modelix.core/model-api/src/jsTest`.

Could you move the test "untyped nodes can be removed" to there?